src/Microsoft.PowerShell.Commands.Utility/commands/utility/Resolve-ErrorRecord.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/create-powershell-scripts.md.https://www.itprotoday.com/powershell/running-powershell-scripts-easy-1-2-3 should work
```powershell
retry of URL failed with error: Network is unreachable
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/powershell-beginners-guide.md.https://www.itprotoday.com/powershell/powershell-basics-remote-management should work
```powershell
retry of URL failed with error: Network is unreachable
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```